plugin.video.nhlgcl/resources/lib/globals.py

```diff
@@ -177,12 +180,12 @@ def get_params():
 def addStream(name,link_url,title,game_id,epg,icon=None,fanart=None,info=None,video_info=None,audio_info=None,teams_stream=None,stream_date=None):
     ok=True
     u=sys.argv[0]+"?url="+urllib.quote_plus(link_url)+"&mode="+str(104)+"&name="+urllib.quote_plus(name)+"&game_id="+urllib.quote_plus(str(game_id))+"&epg="+urllib.quote_plus(str(epg))+"&teams_stream="+urllib.quote_plus(str(teams_stream))+"&stream_date="+urllib.quote_plus(str(stream_date))
-    
+
     if icon != None:
-        liz=xbmcgui.ListItem(name, iconImage="DefaultVideo.png", thumbnailImage=icon) 
+        liz=xbmcgui.ListItem(name, iconImage="DefaultVideo.png", thumbnailImage=icon)
```

Comment:
Can we move this to setart? IconImage and thumbnailImage were deprecated jarvis and up :)
http://mirrors.xbmc.org/docs/python-docs/16.x-jarvis/xbmcgui.html#ListItem

Reply:
Will do.